docs/use-the-network/coverage-mapping/coverage-mapping.mdx

```diff
@@ -86,17 +86,23 @@ Flash your own code and spend your own Data Credits. Setting up a mapper from yo
 * Helium Dev Kit / RAK WisBlock 
     * [Product page via RAKWireless](https://store.rakwireless.com/products/helium-developer-kit), [Product page via ParleyLabs](https://shop.parleylabs.com/collections/iot-developers-products/products/helium-wisblock-connected-kit)
     * [Helium Dev Kit tutorial](https://github.com/arkieguy/RAK4631-Helium-Mapper)
+    * [Make a Helium Mapper with the WisBlock (rakwireless.com)](https://news.rakwireless.com/make-a-helium-mapper-with-the-wisblock/)
 * Heltec CubeCell
-    * [Product page](https://heltec.org/project/htcc-ab02s/)
-    * [Heltec Cubecell tutorial](https://github.com/jas-williams/CubeCell-Helium-Mapper)
+    * [Product page](https://heltec.org/project/htcc-ab02s/), [Product page via Parley Labs](https://shop.parleylabs.com/collections/iot-developers-products/products/cubecell-gps-6502-by-heltec-htcc-ab02s), [IoWE DIY Kit](https://www.internetofwe.net/product-page/diy-coverage-mapper-kit-us915)
+    * [Heltec Cubecell tutorial 1](https://github.com/jas-williams/CubeCell-Helium-Mapper)
```

Comment:
Rather than listing `tutorial 1`, `2`, etc, lets call them out by the code author. That way when we say, "Try the kicko code from the readme" its clear where they go.

Reply:
OK what's your thought on using discord user name vs GitHub user?

Reply:
Only oddity there is that sometimes the discord handle is drastically different than the GitHub username. I think sticking to GH is probably safer so people don't think they're being pointed to some suspect project. I'm open to both! your discretion